strawberryfields/__init__.py

```diff
@@ -31,6 +31,8 @@
 from .tdm import TDMProgram
 from .plot import plot_wigner, plot_fock, plot_quad
 from . import tdm
+from .devicespec import DeviceSpec
+from .result import Result
```

Comment:
Should we add these to `__all__` as well?

Reply:
I was thinking about that as well. Actually, now when you mention it, I think we probably should!